src/reanimated2/animation/decay/utils.ts

```diff
@@ -1,22 +1,19 @@
 'use strict';
-import type { AnimationObject, AnimatableValue } from '../../../index.web';
-import { isWeb } from '../../../reanimated2/PlatformChecker';
 import type {
-  Timestamp,
+  AnimatableValue,
+  AnimationObject,
   Animation,
   ReduceMotion,
+  Timestamp,
 } from '../../../reanimated2/commonTypes';
+import { isWeb } from '../../PlatformChecker';
 
 const IS_WEB = isWeb();
-
 export const VELOCITY_EPS = IS_WEB ? 1 / 20 : 1;
 export const SLOPE_FACTOR = 0.1;
 
-export interface InnerDecayAnimation
-  extends Omit<DecayAnimation, 'current'>,
-    AnimationObject {
-  current: number;
-}
+export type DecayConfig = Partial<Omit<DefaultDecayConfig, 'rubberBandFactor'>>;
+export type WithDecayConfig = DecayConfig;
 
 export interface DecayAnimation extends Animation<DecayAnimation> {
```

Comment:
Well, I cant move `InnerDecayAnimation`, to avoid circular dependencies 🔁 (it is used both in `rigidDecay.ts` and `RubberBand.ts`
Also now, after some improvements I can't move back `DefaultDecayConfig` (for the same reason) and `DecayConfig` neither.